superset/config.py

```diff
@@ -758,11 +758,6 @@ class D3Format(TypedDict, total=False):
 # note: index option should not be overridden
 CSV_EXPORT = {"encoding": "utf-8"}
 
-# Excel Options: key/value pairs that will be passed as argument to DataFrame.to_excel
-# method.
-# note: index option should not be overridden
-EXCEL_EXPORT = {"encoding": "utf-8"}
```

Comment:
The `EXCEL_EXPORT` could include additional keyword arguments other than `encoding`. The challenge is if any of the other [DataFrame.to_excel](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_excel.html) arguments changed, then this PR would be deemed a breaking change and thus we would need to punt on this PR until Superset 4.0.

Reply:
Thank you for your feedback!